data/sql/updates/pending_db_world/rev_1620836710197850400.sql

```diff
@@ -0,0 +1,18 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1620836710197850400');
+UPDATE `creature_template` SET `AIName`="SmartAI" WHERE `entry`=23786;
```

Comment:
Maybe it would be good to verify that it doesn't have a C++ script or it would not be necessary?

```sql
UPDATE `creature_template` SET `AIName`="SmartAI", `ScriptName`="" WHERE `entry`=23786;
```

Or maybe it's not necessary, but just to make sure it doesn't have anything.

Reply:
Couldn't find any C++ script, those are mostly reserved for creatures with many tasks mostly within dungeons and raids. It's always good to make sure though!

Reply:
> Couldn't find any C++ script, those are mostly reserved for creatures with many tasks mostly within dungeons and raids. It's always good to make sure though!

Correct. Just in case. Because in case I had it, I think C++ has priority. But anyway, it was a doubt, it was not a statement. I usually leave it empty.